src/lib/marlin_plonk_bindings/stubs/src/tweedle_fp_urs.rs

```diff
@@ -0,0 +1,171 @@
+use crate::tweedle_dee::{
```

Comment:
Why are the srses special?

Reply:
This mirrors what we did in the previous bindings, where we have 1 long-lived SRS and its references are used in any keys loaded with it. IIRC it's several MBs too, so not ideal to store in the OCaml heap as a custom blob.